ProjectCoimbra.UWP/DataAccessLibrary/DataAccess.cs

```diff
@@ -95,10 +87,22 @@ public static List<string> GetAllData()
         /// Checks if specified name exists in User table
         /// </summary>
         /// <param name="name">The name.</param>
-        /// <returns></returns>
+        /// <returns>true if name exists in local table</returns>
         public static bool Exists(string name)
         {
             return GetAllData().Contains(name);
         }
+
+        private static SqliteDataReader ExecuteCommand(SqliteCommand command)
+        {
+            return command.ExecuteReader();
+        }
+
+        private static string databaseName = "coimbra.db";
+
+        private static string dbpath = Path.Combine(ApplicationData.Current.LocalFolder.Path, databaseName);
```

Comment:
This string and the one below can be made `static readonly` to improve efficiency by reducing the number of instances within the application.

Reply:
Oh yes, forgot to add readonly, thanks!